api/ccapi/include/tensor_dim.h

```diff
@@ -58,13 +58,19 @@ class TensorDim {
     FP32   /** single precision */
   };
 
+  /**
+   * @brief Tensor Data Storage Order. Row-major or Column-major
+   *
+   */
+  enum class StorageOrder { ROW_MAJOR, COL_MAJOR };
```

Comment:
I have a question. Is this differ from HW / WH order?

Reply:
I'm not sure what you mean by HW / WH order.

To explain, row-major and column-major are the ways to store multidimensional arrays.
For instance, we have a matrix A, 

$$ A = \begin{bmatrix}
1 & 2 & 3\\
4 & 5 & 6
\end{bmatrix} $$

If $A$ is stored in row-major, the matrix values in the memory are $[1,2,3,4,5,6]$
If $A$ is stored in column-major, the matrix values in the memory are $[1,4,2,5,3,6]$

Hope this clarifies!

Reply:
OK. it's just storage type.